### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by pmeinhardt**
Just updated all dependencies to their latest versions.

The extension still works as before (checked in Firefox & Chrome):

preview

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -10,12 +10,12 @@ jobs:
...
       - checkout
       - restore_cache:
-          key: dependency-cache-{{ checksum "package.json" }}
+          key: dependency-cache-{{ checksum "yarn.lock" }}-{{ arch }}
       - run:
           name: Install dependencies
           command: yarn install --frozen-lockfile
       - save_cache:
-          key: dependency-cache-{{ checksum "package.json" }}
+          key: dependency-cache-{{ checksum "yarn.lock" }}-{{ arch }}
```
Comment (nickgnd) [inline .circleci/config.yml]: 💡 

Thanks for the explanation in the commit message

File: package.json [hunk 2]
```diff
@@ -32,6 +31,7 @@
     "babel-jest": "^23.6.0",
     "babel-loader": "8.0.5",
     "chrome-launcher": "0.10.5",
+    "clean-webpack-plugin": "^2.0.1",
```
Comment (nickgnd) [inline package.json]: 👍

File: jest.config.js [hunk 3]
```diff
@@ -4,7 +4,7 @@
 module.exports = {
   coverageDirectory: 'coverage',
   collectCoverageFrom: ['<rootDir>/src/**/*.{js,jsx}'],
-  setupTestFrameworkScriptFile: '<rootDir>/test/setup.js',
+  setupFilesAfterEnv: ['<rootDir>/test/setup.js'],
```
Comment (nickgnd) [inline jest.config.js]: Is it a new jest property? 🤔

Comment (pmeinhardt) [reply jest.config.js]: Yep. The previous one, which accepted only a single path, was deprecated in favor of this new options that allows you to pass an array of paths.

For reference, see the note on:
https://jestjs.io/docs/en/configuration.html#setupfilesafterenv-array

**Comment by pmeinhardt** [comment]
Thanks a lot for your review @nickgnd ❤️

**Pull request merged**